static/js/brand-store/brand-store.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import * as Sentry from "@sentry/react";
```

Comment:
Ignore me if it's not relevant, but isn't loading the entire library bad practice?

Reply:
We're only loading the react part of sentry which we need in order for it to work.